### PR TITLE
docs(v2): add Dart Code Metrics site to showcase page

### PR DESCRIPTION
**Pull request by incendial**
## Motivation

The documentation of Dart Code Metrics is using Docusaurus v2.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan

Run yarn run start:v2 and check that the page http://localhost:3000/showcase is rendered properly.

## Related PRs

None

### CODE REVIEW

File: website/src/data/users.js [hunk 1]
```diff
@@ -314,6 +314,14 @@ const Users = [
     source: 'https://github.com/MisterFISHUP/covid-19-in-france',
     tags: ['opensource', 'i18n', 'large'],
   },
+  {
+    title: 'Dart Code Metrics',
+    description: 'Static analysis tool that helps analyse and improve Dart code quality.',
+    preview: require('./showcase/dart-code-metrics.png'),
+    website: 'https://dartcodemetrics.dev/',
```
Comment (prabodhmeshram) [inline website/src/data/users.js:321]: Just wanted to understand the reason behind adding a forward slash in the url

Comment (incendial) [reply website/src/data/users.js:321]: Copy pasted the url from the browser url bar. Is it wrong to have it?

**Review by lex111 — APPROVED** [review]

**Comment by lex111** [comment]
@incendial it's all right, thanks!

File: website/src/data/users.js (hunk 1 above)
Comment (prabodhmeshram) [reply website/src/data/users.js:321]: No just wanted to know, the reason. Both url works the same way, if URLs are being used as a Baseurl in the code somewhere, it depends if they want to join the URI fragments with a slash or without.

**Pull request merged**